articles/extensions/authorization-extension/v2/implementation/setup.md

```diff
@@ -139,9 +139,4 @@ Then click the **CREATE PERMISSION** button. Then enter the name of the permissi
 
 ![Create Permission](/media/articles/extensions/authorization/create-permission.png)
 
-Once you have your permissions created, you can associate them with [Roles](#roles).
-
-<%= include('./_stepnav', {
- prev: ["Install the Extension", "/extensions/authorization-extension/v2/implementation/installation"],
- next: ["Setup the Extension", "/extensions/authorization-extension/v2/implementation/configuration"]
-}) %>
+Once you have your permissions created, you can associate them with [Roles](#roles).
```

Comment:
Since this custom navigation pointing to the next tutorial was removed, I think it'd be best if you added a next-steps pointing to the "Configure the Extension" step